charts/nlk/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+appVersion: 0.1.0
+description: NGINX LoadBalancer Kubernetes
+name: nginx-loadbalancer-kubernetes
+home: https://github.com/nginxinc/nginx-loadbalancer-kubernetes
+icon: https://raw.githubusercontent.com/nginxinc/nginx-loadbalancer-kubernetes/main/nlk-logo.svg
+keywords:
+- nginx
+- loadbalancer
+- ingress
+kubeVersion: '>= 1.22.0-0'
+maintainers:
+- email: kubernetes@nginx.com
```

Comment:
@brianehlert  is this an acceptable value for the maintainers email?

Reply:
Hi @osokin , we'll need to pull this email address out of here. I think for now let's use the `name` field, one each for @ciroque and @chrisakker

```
maintainers:
   name: "@ciroque"
   name: "@chrisakker"
```

Thanks!